DevAAC/routes/players.php

```diff
@@ -514,6 +514,30 @@
     if($player)
         throw new InputErrorException('Player with this name already exists.', 400);
 
+    if (file_exists(TFS_ROOT . '/data/monster/monsters.xml')) {
+        $xml = simplexml_load_file(TFS_ROOT . '/data/monster/monsters.xml');
+        if (property_exists($xml, 'monster'))
+            $forbiddenPlayerNames = array_map('strtolower', array_column(xml2array($xml)['monster'], 'name'));
+    }
+
+    if ($npcDir = opendir(TFS_ROOT . '/data/npc')) {
```

Comment:
I'd prefer to use https://secure.php.net/manual/en/function.glob.php instead of opendir/readdir/closedir